src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -268,11 +271,11 @@ export class VegaBaseView {
   // space and leave enough space to show the bottom view (the events vis).
   // Ref: https://vega.github.io/vega-lite/docs/size.html#limitations
   addPointInTimeEventPadding(view) {
-    // TODO: 100 is enough padding for now. May need to adjust once the current scrolling/overflow
-    // issue is handled. See https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3501
     const eventVisHeight = 100;
```

Comment:
I see the TODO is removed, but this now seems like a magic number - it would be hard to know in the future whether it can be safely changed and how it was determined.

Reply:
I'll add a comment on the reasoning here. This is the pixel height of the event canvas such that there is sufficient but minimal space to render the event marks. It is correlated to the mark size.

Reply:
added